src/main/java/tc/oc/pgm/commands/ChatCommands.java

```diff
@@ -83,6 +83,12 @@ public void direct(MatchPlayer sender, Player receiver, @Text String message) {
         message,
         "[" + ChatColor.GOLD + "DM" + ChatColor.WHITE + "] {0}: {1}",
         viewer -> viewer.getBukkit().equals(receiver));
```

Comment:
Could probably just change this predicate to match both sender & receiver

Reply:
That is actually what I did at first, although the message doesn't make too much sense then in the perspective of the sender

Me: does /msg yikes woah
[DM] ThatOneTqnk: woah